src/static/config.json

```diff
@@ -1,6 +1,6 @@
 {
     "service_routes": {
-        "auth": "/auth/api/V2",
+        "auth": "/auth",
```

Comment:
Change the service route such as to encompass the _whole_ auth2 service (API&UI endpoints as desc in the auth2 docs). We may want to consider adding an auth2 api endpoint for logout/token-invalidation.